test/evergreen/install_gcp_dependencies.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Install dependencies for the GCP SDK. This method is being used instead of having them installed 
+# on system to reduce any versioning problems in the future if the GCP SDK requirements were to
+# change. The installation of the libraries were followed from GCP SDK's documentation:
+# https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md
+#
+install_abseil ()
+{
+    mkdir abseil-cpp && cd abseil-cpp
+    curl -sSL https://github.com/abseil/abseil-cpp/archive/20230125.0.tar.gz -o abseil-cpp.tar.gz
+    tar --strip-components=1 -xzf abseil-cpp.tar.gz
+    sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
+    mkdir cmake-out && cd cmake-out
+    $CMAKE \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DABSL_BUILD_TESTING=OFF \
+        -DBUILD_SHARED_LIBS=yes ../.
+    make -j 4
+    cd ..
+    sudo /opt/cmake/bin/cmake --build cmake-out --target install
+    cd ..
+}
+
+install_nlohmann ()
+{
+    mkdir json && cd json
+    curl -sSL https://github.com/nlohmann/json/archive/v3.11.2.tar.gz -o nlohmann-json.tar.gz
+    tar --strip-components=1 -xzf nlohmann-json.tar.gz
+    mkdir cmake-out && cd cmake-out
+    $CMAKE \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DBUILD_SHARED_LIBS=yes \
+        -DBUILD_TESTING=OFF \
+        -DJSON_BuildTests=OFF ../.
+    make -j 4
+    cd ..
+    sudo /opt/cmake/bin/cmake --build cmake-out --target install
+    cd ..
+}
+
+install_crc32c ()
+{
+    mkdir crc32c && cd crc32c
+    curl -sSL https://github.com/google/crc32c/archive/1.1.2.tar.gz -o crc32c.tar.gz
+    tar --strip-components=1  -xzf crc32c.tar.gz
+    mkdir cmake-out && cd cmake-out
+    $CMAKE -DCMAKE_BUILD_TYPE=Release \
+        -DBUILD_SHARED_LIBS=yes \
+        -DCRC32C_BUILD_TESTS=OFF \
+        -DCRC32C_BUILD_BENCHMARKS=OFF \
+        -DCRC32C_USE_GLOG=OFF ../.
+    make -j 4
+    cd ..
+    sudo /opt/cmake/bin/cmake --build cmake-out --target install
+    cd ..
+}
+
+if [ "$#" -ne 1 ]; then
+    echo "Usage: install_gcp_dependencies.sh <cmake_path>"
+    exit 1
+fi
+
+CMAKE=$1
```

Comment:
The idea is behind this is so that developers who want to develop on GCP can also run the script by passing in a cmake argument. What are your thoughts?

Reply:
It's a little odd for interactive use but I get why it needs to be this way. If we're going to do it like this, can you please fix the `sudo /opt/cmake/bin/cmake` call to use the `$CMAKE` that got passed in?

Reply:
Thanks for pointing it out here [2aceb41](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/2aceb41bdf890876e4c4406b1c94d5dcec76360f)

Reply:
Thanks for pointing that out! done!